### DISCUSSION TITLE
FCOS kernel version vs kernel-header version

### DISCUSSION THREAD

**Discussion by itmwiw** [okd-project/okd#1031] [Q&A]
Hello,
I'm trying to use the nvidia gpu operator on OKD 4.9.0-0.okd-2021-12-12-025847. I am facing the following issue: Actually gpu-operator search for kernel-header version based on Host's kernel version. The problem is that the  OKD 4.9.0-0.okd-2021-12-12-025847's version install kernel version 5.14.14 wich does not have an equivalent kernel-header version:
* the kernel 5.14.14 exists: https://kojipkgs.fedoraproject.org/packages/kernel/
* the kernel-header 5.14.14  does not exists : https://kojipkgs.fedoraproject.org/packages/kernel-headers/
So my question is: can I update the kernel for a specific OKD version without changing the whole cluster version (using machineconfig for example?)
Thanks in advance
Tarik

**Comment by vrutkovs** [comment]
>kernel version 5.14.14 wich does not have an equivalent kernel-header version

This is Fedora's issue, please file a bugzilla for kernel component in Fedora
